corehq/apps/app_manager/views/forms.py

```diff
@@ -721,14 +723,24 @@ def get_form_view_context_and_template(request, domain, form, langs, current_lan
         valid_index_names.append(USERCASE_PREFIX[0:-1])     # strip trailing slash
 
     form_has_schedule = isinstance(form, AdvancedForm) and module.has_schedule
+
+    try:
+        case_properties_map = get_all_case_properties(app)
+        usercase_properties_map = get_usercase_properties(app)
+    except CaseError as e:
+        case_properties_map = {}
+        usercase_properties_map = {}
```

Comment:
Is it likely that only one of these calls will fail?  Would it be useful to let the other go through, or is that unimportant?

Reply:
I think that if either of them error, your app needs to be fixed before you can meaningfully use the page, and you'll need to refresh after fixing it to correctly populate the properties map, so I'm not worried about making the page all that useful while you're in the error state.